client/assets/langs/ko.json

```diff
@@ -40,11 +41,13 @@
+  "LIMIT_FILE_SIZE": "100MB 이하의 파일만 업로드할 수 있습니다",
```

Comment:
How about passing `{{videoFileSize}}` param after adding const variable in `const.ts`?